docker-compose.yml

```
@@ -44,34 +28,6 @@ services:
 volumes:
-  wanderer_data:
-  postgres_data:
-    external: true
-
-networks:
-  notifier-internal:
-    internal: true
-  web:
-    external: true
\ No newline at end of file
+  wanderer_data:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)